src/google/protobuf/compiler/command_line_interface.cc

```diff
@@ -1771,10 +1771,23 @@ bool CommandLineInterface::MakeInputsBeProtoPathRelative(
 
 
 bool CommandLineInterface::ExpandArgumentFile(
-    const std::string& file, std::vector<std::string>* arguments) {
+    const char* file, std::vector<std::string>* arguments) {
+
+// On windows to force ifstream to handle proper utr-8, we need to convert to proper supported utf8 wstring.
+// If we dont then the file can't be opened.
+#ifdef _MSC_VER
+  // Convert the file name to wide chars.
+  int size = MultiByteToWideChar(CP_UTF8, 0, file, strlen(file), NULL, 0);
+  std::wstring fileStr;
+  fileStr.resize(size);
+  MultiByteToWideChar(CP_UTF8, 0, file, strlen(file), &fileStr[0], fileStr.size());
+#else
+  std::string fileStr(file);
```

Comment:
nit: file_str to adhere to the google style for variable naming

Reply:
Fixed https://github.com/protocolbuffers/protobuf/pull/14253/commits/cfbdd357cbed555f8a09d186dbc1f26943002c1c